utest/ui/ErrorMinimizers.cpp

```diff
@@ -34,14 +34,29 @@ class ErrorMinimizerTest: public IcpHelper
 
 TEST_F(ErrorMinimizerTest, PointToPointErrorMinimizer)
 {
-	setError("PointToPointErrorMinimizer");	
+	setError("PointToPointErrorMinimizer");
```

Comment:
Those kind of tests are now handled by adding a yaml file in libpointmatcher/examples/data/icp_data/

Here is the info I could find about that in a pull request:

> The idea here is that when a new test is added, or existing behavior is modified, the developer does not have to edit a lot of code. Just data files on disk need to be added or updated (and checked in). For example, say a new yaml file is added, testing a new filter. Running the tests will make the framework iterate over all yaml files, including the new one. The "current" ICP transform for the new yaml file will be written to disk. There was no reference (known true) ICP transform yet to compare with, so the test for the current yaml file will fail. Then copy the newly created "current" ICP transform over the future "reference" (known as correct) ICP transform file. This time when the tests are run they will succeed as the two data files have identical data. Same logic is used if existing behavior is modified.

I should add that in the unit test tutorial at some point...